jobrunner/schema.py

```diff
@@ -0,0 +1,43 @@
+from dataclasses import dataclass
+from enum import Enum
+
+from jobrunner.job_executor import ExecutorState
+
+
+class TaskType(Enum):
+    RUNJOB = "runjob"
+    # TODO: delete job
+
+
+class TaskStage(Enum):
```

Comment:
should this be called ExecutorState for the time being?

Reply:
I don't think so, as we need to add a delete file task for step 1, i.e. soon